cfpb/cfgov-refresh
# Upgrade mapbox to version 3.2.0

Our implementation of Mapbox in [Find a Housing Counselor](https://www.consumerfinance.gov/find-a-housing-counselor/) is not adding proper attribution. Upgrading to version `3.2.0` fixes this.

## Changes
- Upgrade mapbox (in Find a Housing Counselor app) to version 3.2.0

## Testing
1. Startup `cfgov-refresh`
2. Go to (http://localhost:8000/find-a-housing-counselor/)
3. Look for the attribution (as seen in Screenshot below)
4. Make sure the app works! Maybe try some different browsers, too!

## Screenshots
<img width="1192" alt="screen shot 2019-03-08 at 12 04 50 pm" src="https://user-images.githubusercontent.com/1490703/54043813-7d557680-419b-11e9-9028-2c9b59678793.png">

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
